webpack.config.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const path = require("path");
+const CleanCSSPlugin = require('less-plugin-clean-css');
+
+module.exports = {
+    entry: {
+        demo: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/demo/index.js")],
+        parser: ["core-js/stable", "regenerator-runtime/runtime", path.join(__dirname, "src/js/parser/index.js")]
+    },
+    output: {
+        path: path.join(__dirname, "assets/build/js"),
+        filename: "[name].js"
+    },
+    module: {
+        rules: [
+            {
+                test: /\.jsx?$/,
+                loaders: ["babel-loader"],
+                include: path.join(__dirname, "src/js"),
+                exclude: /node_modules|src\/js\/vendor/
+            },
+            {
+                test: /\.less$/,
```

Comment:
Letting Webpack bundle these styles with their relevant bundles seemed like an easier way to do this (especially now that demo also requires the CodeMirror styles). Thoughts?